src/coordinator_handler/types.rs

```diff
@@ -13,13 +13,60 @@ pub struct Response<T> {
     pub data: Option<T>,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum ProverType {
+    Undefined,
+    Chunk,
+    Batch,
+    OpenVM,
+}
+
+impl ProverType {
+    pub fn from_u8(v: u8) -> Self {
+        match v {
+            1 => ProverType::Chunk,
```

Comment:
Notice that the `ProverType` here is not completly match with which has been defined in `prover/types`. Does it suppose that any "OpenVM" prover would support all types of proving (chunk/batch/bundle)?

Reply:
In practice, yes they will. But it is still possible to selectively support subset of proof types in a openvm prover. It is possible to only ask for certain types of tasks in a GetTask request. https://github.com/scroll-tech/scroll-proving-sdk/pull/62/files#diff-5b38dca990c13c0d1398729aebaaf85541958a4753de6ec938955b11a790cf7fL44-L47